### ISSUE TITLE
always improving documentation

### ISSUE THREAD

**Issue by jlwaugh** [enigmampc/SecretNetwork#304]
Continually enhancing the docs 📑

Learn pages (general community):
[learn.scrt.network](https://learn.scrt.network) ~ [GitHub repo](https://github.com/SecretFoundation/SecretWiki)

Build pages (developers and node operators):
[build.scrt.network](https://build.scrt.network) ~ [GitHub repo](https://github.com/enigmampc/SecretNetwork/tree/master/docs)

**Comment by cankisagun** [comment]
For week of 6/29:
- focus is on the learn page
- once this is done we should focus on contribution guide - #176 (this should be done for Enigma and Secret Foundation repos)
- after those two tasks we will announce the wiki (6/30 or 7/1)
- build page is waiting for contract guide + secretJS

cc @jlwaugh

**Comment by cankisagun** [comment]
For the sprint of 7/6- 7/17
Focus is primarily on the build section of documentation:
- [ ] add secret contract development guide
- [ ] local environment setup guide
- [ ] create contract catalog using hackaton sbumissions

**Comment by cankisagun** [comment]
For the week of 7/20
- [ ] Priority - local environment setup for development guide (secret counter uses this)
- [ ] Priority - testnet setup for development guide (secret counter uses this)
- [ ] Low priority- add secret contract development guide (getting started guide)
- [ ] Low priority- create contract catalog using hackaton sbumissions
